modules/gapi/perf/common/gapi_imgproc_perf_tests.hpp

```diff
@@ -41,6 +41,12 @@ class CannyPerfTest           : public TestPerfParams<tuple<compare_f, MatType,c
 class GoodFeaturesPerfTest    : public TestPerfParams<tuple<compare_vector_f<cv::Point2f>, std::string,
                                                             int,int,double,double,int,bool,
                                                             cv::GCompileArgs>> {};
+class BoundingRectMatPerfTest       :
+    public TestPerfParams<tuple<compare_rect_f, MatType2,cv::Size,bool, cv::GCompileArgs>> {};
```

Comment:
Can you use `CompareRects` here instead of `compare_rect_f`?

Reply:
Decided that yes, we can. Will create an issue to refactor all the other perf.tests like that

Reply:
Ok.